app/imports/templates/components/nameStatus.js

```diff
@@ -13,6 +13,13 @@ Template['components_nameStatus'].onCreated(function() {
     try {
       registrar.getEntry(name, (err, entry) => {
         if(!err && entry) {
+          let prevInfo = TemplateVar.get(template, 'nameInfo');
```

Comment:
There's stuff below that line that would be undesirable to run unless the name or status changes (The resetting of the TemplateVars, which cause a flicker if they're reset every second).